projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -18,79 +23,198 @@ npm install @ngrx/effects --save
+export class MoviesService {
+  constructor (private http: HttpClient) {}
+
+  getAll() {
+    return this.http.get('/movies');
+  }
+}
+</code-example>
+
+The component has multiple responsibilities:
+
+- Managing the _state_ of the movies.
+- Using the service to perform a _side effect_, in reaching out to an external API to fetch the movies
+- Changing the _state_ of the movies within the component.
+
+`Effects` when used along with `Store`, decrease the responsibility of the component.  In a larger application, this becomes more important as you have multiple sources of data, with multiple services required to fetch those pieces of data, and services potentially relying on other services.
+
+Effects are where external data and interactions are handled, allowing your services to be less stateful, and simply performers of tasks related to external interactions. Refactoring the component to put the shared movie data in the `Store`, Effects are where the fetching of movie data is handled.
+
+<code-example header="movies-page.component.ts">
+@Component({
+  template: `
+    &lt;div *ngFor="let movie of movies$ | async"&gt;
+      {{ movie.name }}
+    &lt;/div&gt;
+  `
+})
+export class MoviesPageComponent {
+  movies$: Observable<Movie[]> = this.store.select(state => state.movies);
+
+  constructor(private store: Store&lt;{ movies: Movie[] &gt;}) {}
+
+  ngOnInit() {
+    this.store.dispatch({ type: '[Movies Page] Load Movies' });
+  }
+}
+</code-example>
+
+The movies are still fetched through the `MoviesService`, but the component is no longer concerned with how the movies are fetched and loaded. Its only responsible for declaring its _intent_ to load movies, and using selectors to access movie list data. Effects are where the asynchronous activity of fetching movies happens. Your component becomes easier to test, and less responsible for the data it needs.
 
-Effects are injectable service classes that use two main APIs.
+## Writing Effects
 
-### Effect decorator
+To isolate side-effects from your component, you must create an `Effects` class to listen for events and perform tasks. 
 
-The `Effect` decorator provides metadata to register observable side-effects in the effects class. Registered effects provide new actions provided by the source Observable to the store.
+Effects are injectable service classes with distinct parts:
 
-### Actions Observable
+- An injectable `Actions` service that provides an observable stream of _all_ actions dispatched _after_ the latest state has been reduced.
+- Observable streams are decorated with metadata using the `Effect` decorator. The metadata is used to register the streams that are subscribed to the store. Any action returned from the effect stream is then dispatched back to the `Store`.
+- Actions are filtered using a pipeable `ofType` operator. The `ofType` operator takes one more action types as arguments to filter on which actions to act upon.
+- Effects are subscribed to the `Store` observable. 
```

Comment:
would it be more correct to use `Actions` observable here??

```suggestion
- Effects are subscribed to the `Actions` observable. 
```

Reply:
Store is more correct here. I'm referring to how the actions get dispatched back to the store.